src/build_scripts/buildlib_cmake.internal_components

```diff
@@ -7,7 +7,7 @@ components.
 
 import sys, os
 
-_CIMEROOT = os.path.join(os.path.dirname(os.path.abspath(__file__)), "..","..","..","..", "..")
+_CIMEROOT = os.environ.get("CIMEROOT")
```

Comment:
I think we should add an error message here for the case when CIMEROOT is not defined.